src/analysis/ml.py

```diff
@@ -72,8 +70,7 @@ def summarize_networks(file_paths: Iterable[Union[str, PathLike]]) -> pd.DataFra
 
     return concated_df
 
-
-def pca(dataframe: pd.DataFrame, output_png: str, output_file: str, output_coord: str):
+def pca(dataframe: pd.DataFrame, output_png: str, output_file: str, output_coord: str, components=2, labels=True):
```

Comment:
Can specify types for the new arguments and add them to the docstring

Reply:
specify them as a new name?